packages/app-bridge/src/AppBridgeBlock.ts

```diff
@@ -169,6 +171,10 @@ export interface AppBridgeBlock<
 
     getCurrentLoggedUser(): Promise<User>;
 
+    /**
+     * @deprecated This will be removed in version 4.0.0 of @frontify/app-bridge
+     * Use appBridge.api({ name: 'getAssetBulkDownloadToken', payload: { settingIds, documentBlockId: appBridge.getBlockId() },}); instead
+     */
```

Comment:
Not sure if needed as the hook and this function was never announced

Reply:
Either you deprecate, either you delete it fully, up to you guys ;)

Reply:
We will decide within the team next week, thanks

Reply:
the old implementation is deleted